### PR TITLE
docs: apply style updates to Kafka Getting Started quick start

### PR DESCRIPTION
**Pull request by jbyrne-redhat**
**Notes for reviewers**

- Applies [MAS style guidelines](https://docs.google.com/document/d/1TERH4XRH6La44hUQH88LUaax-35JH3dLXmceGJoLv7I/edit#) to the Kafka _Getting Started_ quick start
- Reflects some of the style updates to the Kafka _Quarkus_ and _Node.js_ quick starts
- An HTML preview build of the Getting Started quick start is available for review [here](http://file.emea.redhat.com/~jbyrne/JK-9560/README.html)

### CODE REVIEW

File: docs/kafka/getting-started-kafka/README.adoc [hunk 1]
```diff
@@ -228,24 +217,24 @@ ifdef::qs[]
...
+* You have a running Kafka instance in {product-kafka}.
 * You've created a service account that you want to allow to access the running Kafka instance.
 
 .Procedure
-. In the *Kafka Instances* page of the web console, click the name of the Kafka instance that you want the service account to access.
+.  In the {product-long-kafka} {service-url-kafka}[web console^], select *Kafka Instances* and then click the name of the Kafka instance that you want the service account to access.
 . Click the *Access* tab to view the current ACL for this instance.
-. Click *Manage access*, use the *Account* drop-down menu to select the service account that you previously created, and click *Next*.
-. Under *Assign Permissions*, use the drop-down menu to select the *Consume from a topic* and the *Produce to a topic* permission options, and set all resource identifiers to `Is` and all identifier values to `*`.
+. Click *Manage access*, use the *Account* list to select the service account that you previously created, and click *Next*.
```
Comment (AedinC) [inline docs/kafka/getting-started-kafka/README.adoc:236]: I'd suggest breaking this step into 2 as after they click Manage access, a new dialog box opens so it's not really part of the same step.

File: docs/kafka/quarkus-kafka/README.adoc [hunk 2]
```diff
@@ -177,13 +177,13 @@ endif::[]
 == Creating a Kafka topic in {product-kafka}
 
 [role="_abstract"]
-For this quick start, the Kafka topic that the Quarkus example application references is called `prices`. You need to create this topic in {product-long-kafka} so that the Quarkus application can interact with it.
+The Quarkus application in this quick start uses a Kafka topic called `prices` to produce and consume messages. In this task, you'll create the topic in your Kafka instance.
```
Comment (AedinC) [inline docs/kafka/quarkus-kafka/README.adoc:180]: Suggest slight rewrite of  'In this task, you'll create the 'prices' topic in your Kafka instance.' just to remove any ambiguity.

**Review by AedinC — APPROVED** [review]
Really strong set of updates John. Just a couple of minor suggestions.

File: docs/kafka/getting-started-kafka/README.adoc (hunk 1 above)
Comment (jbyrne-redhat) [reply docs/kafka/getting-started-kafka/README.adoc:236]: Yes, good call.

**Pull request merged**